src/docs/codelabs/implicit-animations.md

```diff
@@ -0,0 +1,472 @@
+---
+title: "Implicit animations"
+description: "A codelab that uses interactive examples and exercises to teach  how to use Flutter's implicitly animated widgets."
+toc: true
+diff2html: true
+js:
+- url: /assets/codelabs/js/animations_examples.js
+  defer: true
+---
+
+<?code-excerpt path-base="animation/implicit"?>
+
+Welcome to the implicit animations codelab, where you learn how to use Flutter
+widgets that make it easy to create animations for a specific set of properties.
+
+To get the most out of this codelab, you should have basic knowledge about:
+- How to [make a Flutter app].
+- How to use stateful widgets.
+
+This codelab covers the following material:
+- Using `AnimatedOpacity` to create a fade-in effect.
+- Using `AnimatedContainer` to animate transitions in size, color, and margin.
+- Overview of implicit animations and techniques for using them.
+
+**Estimated time to complete this codelab: 15-30 minutes.**
+
+{{site.alert.important}}
+  This page uses an embedded version of [DartPad] to display examples and exercises.
+  If you see empty boxes instead of DartPads, go to the
+  [DartPad troubleshooting page].
+{{site.alert.end}}
+
+## What are implicit animations?
+
+With Flutter's animation library, you can add motion and create visual effects
+for the widgets in your UI. One part of the library is an assortment of widgets
+that manage animations for you. These widgets are collectively referred to as _implicit
+animations_, or _implicitly animated widgets_, deriving their name from the
+[ImplicitlyAnimatedWidget] class that they implement. Implicit animations trade
+control for convenience&mdash;they manage animation effects so that you don't have to.
```

Comment:
I always like to say that with implicit animations you you set a target value and when that target value changes the widget animates you from the old to the new.

Reply:
Perfect! This adds some much-needed glue the intro was missing. Many thanks.